OmsAgent/Fairfax/omsagent_ff.py

```diff
@@ -112,6 +113,10 @@ def main():
             message = 'Install failed with exit code {0}: please install ' \
```

Comment:
The messages logged by the omsagent bundle are not exposed as an extension message, from the perspective that if the extension install fails, these strings are what they will see in the Azure portal.
Most customers do not know where on the machine to find more descriptive logs on the machine, and we should not require them to go to the machine to figure out the next step.